provider/resource_keycloak_realm.go

```diff
@@ -352,6 +352,17 @@ func resourceKeycloakRealm() *schema.Resource {
 				Computed:         true,
 				DiffSuppressFunc: suppressDurationStringDiff,
 			},
+			"oauth2_device_code_lifespan": {
+				Type:             schema.TypeString,
+				Optional:         true,
+				Computed:         true,
+				DiffSuppressFunc: suppressDurationStringDiff,
+			},
+			"oauth2_device_polling_interval": {
```

Comment:
should this attribute also be a duration string?

Reply:
![1](https://user-images.githubusercontent.com/43110475/132201150-603a220d-1227-4abb-b1de-339dd9eb1c15.png)
![2](https://user-images.githubusercontent.com/43110475/132201163-73981160-2ab8-4fde-ac64-3e83915adb43.png)

The polling interval should be set as seconds. There is also no possibility to choose other time units in the UI, see attached screenshot.